database-management/migrations/20180402170153_make-varchar-uniform.js

```diff
@@ -0,0 +1,156 @@
+/* eslint-disable */
+exports.up = async knex => {
+  await knex.schema.alterTable('articles', table => {
```

Comment:
Minor refactor, all of these commands are independent, so you can remove all the awaits in both the up and down, and just chuck them in an array and wrap them in `await Promise.all()` or even just an arrow function with implicit return value as well